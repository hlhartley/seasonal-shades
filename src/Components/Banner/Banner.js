import React from 'react';

export const Banner = () => {
    return(
        <p>Banner</p>
    )
}