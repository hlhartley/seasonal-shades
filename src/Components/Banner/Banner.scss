.banner-container {
    align-items: center;
    display: flex;
    background-color: #efdac0;

    .banner-img {
        max-height: 500px;
        width: 75vw;
    };

    .banner-text {
        text-align: center;
    };
};