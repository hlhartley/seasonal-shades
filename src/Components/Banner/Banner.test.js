import React from 'react';
import Banner from './Banner';
import { shallow } from 'enzyme';

describe('Banner component', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Banner type={'lipstick'}/>)
    })
    it('should match the snapshot with all data passed in correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
