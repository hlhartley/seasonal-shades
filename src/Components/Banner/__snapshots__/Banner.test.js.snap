// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner component should match the snapshot with all data passed in correctly 1`] = `
<div
  className="banner-container"
>
  <img
    alt="colors banner"
    className="banner-img"
    src="lipstick.jpg"
  />
</div>
`;
